src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const max = 10;
+const min = 2;
+
+const setButton = (el, state) => (el.disabled = state);
+
+appendRow.addEventListener('click', (e) => {
+  const rows = [...tbody.querySelectorAll('tr')];
+
+  rows.push(rows[1].cloneNode(true));
+  rows.forEach((row) => tbody.appendChild(row));
```

Comment:
In this block of code, you are appending rows to the table body. However, you are doing so in a loop. This could lead to unnecessary reflows and repaints in the browser, which can degrade performance. It's better to create a document fragment, append your rows to it, and then append the fragment to the table body.